kanidm_book/src/installing_client_tools.md

```diff
@@ -9,7 +9,7 @@ Kanidm currently supports:
  * OpenSUSE Tumbleweed
  * OpenSUSE Leap 15.3/15.4
  * Fedora 34/35
- * Centos Stream 9
+ * Centos Stream 9 (alpha 7 only)
```

Comment:
This wording isn't clear to me - do you mean Kanidm alpha 7, or CentOS?

Reply:
I think they mean alpha 7 only, because that was the first version that would work on centos. So I think in this case because we don't support "older" versions, we don't need this as we can safely assume everyone should be on alpha 8 and above.